packages/cli/src/executions/execution.types.ts

```diff
@@ -69,6 +78,8 @@ export namespace ExecutionSummaries {
 		metadata: Array<{ key: string; value: string }>;
 		startedAfter: string;
 		startedBefore: string;
+		annotationTags: string[]; // tag IDs
```

Comment:
👍 for the comment

Could also be `Array<AnnotationTagEntity['id']>`

Reply:
It's a type for a serialized filter value from a query, so I guess string would fit better here